hack/after-build/update-generated-deep-copies.sh

```diff
@@ -57,6 +57,6 @@ function generate_deep_copies() {
 }
 
 # v1 is in the group ""
-DEFAULT_VERSIONS="/ v1 extensions/ extensions/v1beta1"
+DEFAULT_VERSIONS="/ v1 extensions/ extensions/v1beta1 componentconfig/v1alpha1"
```

Comment:
I see deep copies are generated for types in componentconfig/. How did you do that?